src/hooks/useGetDocumentTitlePrice.ts

```diff
@@ -2,12 +2,17 @@ import { useEffect } from 'react'
 import { usePriceCakeBusd } from 'state/hooks'
 
 const useGetDocumentTitlePrice = () => {
-  const cakePriceUsd = usePriceCakeBusd()
+  const cakePriceUsdNumber = usePriceCakeBusd().toNumber()
```

Comment:
`usePriceCakeBusd` returns a BigNumber, on which the `isNan` function is already implemented https://mikemcl.github.io/bignumber.js/#isNaN
It's a good habit to keep numbers as BigNumbers until we don't have any other choice.

Reply:
so
```
const cakePriceUsd = usePriceCakeBusd()
 const cakePriceUsdString = cakePriceUsd.isNaN()
    ? ''
    : ` - $${cakePriceUsd.toNumber().toLocaleString('en-EN', {
        minimumFractionDigits: 3,
        maximumFractionDigits: 3,
      })}`
```